zhaquirks/danfoss/thermostat.py

```diff
@@ -26,47 +26,171 @@
     OUTPUT_CLUSTERS,
     PROFILE_ID,
 )
-from zhaquirks.danfoss import D5X84YU, DANFOSS
+
+from . import DANFOSS, HIVE, POPP
```

Comment:
For making it easier to use custom quirks, please use absolute imports instead of relative imports.

Reply:
Forgot about that one. I'll fix that.